### ISSUE TITLE
 support of nhwc and ndhwc formats for batchnorm based on mkl dnn api

### ISSUE THREAD

**Issue by shyrma** [deeplearning4j/deeplearning4j#8314]
eventually check whether new releases of mkl dnn library support data format with channel dimension as last axis (format_tag::nhwc/ndhwc batch normalization operation, both forward and backprop),
if so, provide corresponding options in our operation based on mkl dnn api

**Comment by raver119** [comment]
@shyrma i've upgraded MKL-DNN to the latest branch and version today, it might be a good idea to check it out now

**Comment by shyrma** [comment]
I have just checked - still not working :-(

**Comment by agibsonccc** [comment]
Closing — this issue is over 5 years old with no recent activity. The codebase has been significantly rewritten. If this is still relevant, please open a new issue.
